src/universal/decorators/requireAuth/requireAuth.js

```diff
@@ -3,7 +3,7 @@ import {push} from 'react-router-redux';
 import {localStorageVars} from '../../utils/clientOptions';
 
 let key;
-export default ComposedComponent => {
```

Comment:
this is the worst rule that eslint ever made. I say we make a global exception for it, otherwise our HOCs are gonna be ugly